docs/incidents/incident-customize-incident-layout.md

```diff
@@ -76,3 +76,1470 @@ To change the information that appears in dynamic sections:
 
 You can add new sections or fields to the layout by dragging and dropping them from the **Library** on the left into the layout. For example, insert a new field that you created into a new, or already existing, section.
 
+## Layout file structure
+
+Layout structure differs before and after Cortex XSOAR version 6.0.
+
+#### Up to Cortex XSOAR version 6.0
+
+Each layout kind should be represented in one file, as exported from Cortex XSOAR, with addition of the field `"toVersion": "5.9.9"`.
+
+The file should be named `layout-<KIND>-<TYPE>.json`, e.g. `layout-details-Phishing.json`
+
+#### Cortex XSOAR version 6.0 and above
+
+Cortex XSOAR version 6.0 introduces better layout management. As part of it, all the kinds and types (i.e. detailsV2, edit, etc...) of layouts are consolidated in one file, as described below.
+
```

Comment:
Is there need to do anything after exporting the file from Server? Let's add a one liner explaining the export process.

Reply:
basically just needs to add the `fromVersion` as the file already includes all required fields (and more)

we will need to add support to the `format` cmd to add the `fromVersion` and remove garbage fields